makelib/golang.mk

```diff
@@ -166,6 +166,11 @@ go.test.unit: $(GOJUNIT) $(GOCOVER_COBERTURA)
 	@$(GOCOVER_COBERTURA) < $(GO_TEST_OUTPUT)/coverage.txt > $(GO_TEST_OUTPUT)/coverage.xml
 	@$(OK) go test unit-tests
 
+go.test.unit.nocov:
+	@$(INFO) go test unit-tests without coverage
+	@CGO_ENABLED=0 $(GOHOST) test $(ARGS) $(GO_TEST_FLAGS) $(GO_STATIC_FLAGS) $(GO_PACKAGES) || $(FAIL)
```

Comment:
could this be accomplished by adding a new environment variable NOCOV instead of a new target?

Reply:
Totally, I'll update this PR so it checks for the `NOCOV` env var in the existing `go.test.unit` target.